gtsfm/utils/io.py

```diff
@@ -159,6 +165,50 @@ def export_model_as_colmap_text(gtsfm_data: GtsfmData, images: List[Image], save
     write_points(gtsfm_data, images, save_dir)
 
 
+def colmap2gtsfm(
+    cameras: Dict[int, ColmapCamera],
+    images: Dict[int, ColmapImage],
+    points3D: Dict[int, ColmapPoint3D],
+    load_sfmtracks: bool = False,
+) -> Tuple[List[Cal3Bundler], List[Pose3], List[str], Optional[List[Point3]]]:
+    """Converts COLMAP-formatted variables to GTSfM format.
+    Args:
+        cameras: dictionary of COLMAP-formatted Cameras
+        images: dictionary of COLMAP-formatted Images
+        points3D: dictionary of COLMAP-formatted Point3Ds
+        return_tracks (optional): whether or not to return tracks
+    Returns:
+        cameras_gtsfm: list of N camera calibrations corresponding to the N images in images_gtsfm
+        images_gtsfm: list of N camera poses when each image was taken
+        img_fnames: file names of images in images_gtsfm
+        sfmtracks_gtsfm: tracks of points in points3D
```

Comment:
just a nit: it seems more meaningful (to me) to return these in this order: filenames, images, cameras, tracks. @johnwlambert ?

Reply:
any thoughts on this @womackj1 ?